### PR TITLE
Add preview related improvements

### LINKED ISSUE #62: Need ability to generate json for preview/draft items
Would be great to have this feature

### PR DESCRIPTION
**Pull request by dlitvakb**
Fixes #62 
Fixes #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +192,37 @@ a common baseline. You can find a complete example [here](./examples/templates/c
 Using the `--mark-processed` option alongside `--json-template` will mark all resources as `bootstrapProcessed`,
 which will make it so `update_space` calls avoid already created resources. (A resource being either a Content Type, Entry or Asset).
 
+## Workflow for backing up draft/updated content
+
+On many cases, you want to have a full dump of your whole space, including draft/updated content.
+To achieve this, do the following steps:
```
Comment (Roblinde) [inline README.md]: minimal nit, I think it should be ", complete the following steps:"

I don't think you "do"steps, am I right @loudmouth?

Comment (dlitvakb) [reply README.md]: how about `follow the next steps`?

Comment (Roblinde) [reply README.md]: Just noticed it should be "In many cases"... missed that before, sorry 😞

Comment (loudmouth) [reply README.md]: "do the following"

**Pull request merged**